picocolors.js

```diff
@@ -1,12 +1,12 @@
-let tty = require("tty")
-
+let argv = process.argv || [],
+	env = process.env
```

Comment:
btw, shall we also handle scenarios when ```process``` is undefined? (like in the browser)

```env = typeof process !== 'undefined' ? process.env : {};```

or may you like me to make a commit?